test/utils/UintUtils.ts

```diff
@@ -31,11 +31,74 @@ describe('UintUtils', function () {
 
     describe('#toHexString', function () {
       describe('(uint256)', function () {
-        it('todo');
+        it('Should return 0 if input is 0', async () => {
+          const zeroNumber = ethers.BigNumber.from('0');
+          expect(
+            await instance.callStatic['toHexString(uint256)'](zeroNumber),
+          ).to.equal('0x00');
+        });
+
+        it('Should return correct hexString represenation of a number', async () => {
+          const inputValues = [
+            '1000',
+            '1',
+            '12345',
+            '85746201361230',
+            '999983',
+          ];
+          const outputValues = [
+            '0x03e8',
+            '0x01',
+            '0x3039',
+            '0x4dfc57df7b4e',
+            '0x0f422f',
+          ];
+          for (let i = 0; i < inputValues.length; i++) {
+            expect(
+              await instance.callStatic['toHexString(uint256)'](
+                ethers.BigNumber.from(inputValues[i]),
+              ),
+            ).to.equal(outputValues[i]);
+          }
+        });
       });
 
       describe('(uint256,uint256)', function () {
-        it('todo');
+        it('Fails if length input is 0 and value is nonzero', async () => {
```

Comment:
```suggestion
        it('fails if length input is 0 and value is nonzero', async () => {
```
Use lower-case test descriptors (for consistency).

Reply:
Also, I prefer to put revert tests in a `describe('reverts if', function () { ... })` block.